source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cpp

```diff
@@ -39,28 +65,6 @@ int main()
         };
         std::cout << settings.processing() << std::endl;
 
-        std::cout << "Configuring base acquisition with settings same for all HDR acquisition:" << std::endl;
-        const auto baseAcquisition = Zivid::Settings::Acquisition{ Zivid::Settings::Acquisition::Brightness{ 1.8 } };
-        std::cout << baseAcquisition << std::endl;
```

Comment:
Note the comment here ("Configuring base acquisition with settings same for all HDR acquisition:"). The point was to demonstrate that one can set the base acquisition settings (same for all HDR acquisitions) and then modify others. This demonstration is lost in this PR.

Reply:
Reverted to old method, we are now demostrating this again